File: examples/src/Presentations/Conversion/ConversionToTIFFNotes.cs
﻿using Aspose.slides.Export

/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Slides for .NET API reference 
when the project is build. Please check https:#docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Slides for .NET API from http:#www.aspose.com/downloads, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using http:#www.aspose.com/community/forums/default.aspx
*/

namespace Aspose.slides.Examples.CSharp.Presentations.Conversion
{
    class ConversionToTIFFNotes
    {
        public static void Run()
        {
            #ExStart:ConversionToTIFFNotes
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Conversion()

            # Instantiate a Presentation object that represents a presentation file
            using (Presentation presentation = new Presentation(dataDir + "NotesFile.pptx"))
            {
                # Saving the presentation to TIFF notes
                presentation.save(dataDir + "Notes_In_Tiff_out.tiff", SaveFormat.Tiff)
            }
            #ExEnd:ConversionToTIFFNotes
        } 
    }
}

 

File: examples/src/Presentations/Conversion/conversion_to_tiff_notes.py
import aspose.slides as slides


def convert_to_tiff_notes(global_opts):
    # Instantiate a Presentation object that represents a presentation file
    with slides.Presentation(global_opts.data_dir + "presentation_with_notes.pptx") as presentation:
        # Saving the presentation to TIFF notes
        presentation.save(global_opts.out_dir + "convert_to_tiff_notes_out.tiff", slides.export.SaveFormat.TIFF)
